clang/test/ClangScanDeps/include-tree-working-directory.c

```diff
@@ -20,18 +20,18 @@
 // CACHE-HIT: remark: compile job cache hit
 
 // RUN: cat %t/tu.rsp | sed -E 's|.*"-fcas-include-tree" "(llvmcas://[[:xdigit:]]+)".*|\1|' > %t/tu.casid
-// RUN: clang-cas-test -cas %t/cas -print-include-tree @%t/tu.casid | FileCheck %s -DPREFIX=%/t
+// RUN: clang-cas-test -cas %t/cas -print-include-tree @%t/tu.casid | %PathSanitizingFileCheck --sanitize PREFIX=%/t %s
 
-// CHECK: [[PREFIX]]/t.c llvmcas://
+// CHECK: PREFIX{{/|\\}}t.c llvmcas://
```

Comment:
We have a number of tests that do something like pipe the output through `sed 's:\\\\\?:/:g'` to avoid needing regex for every CHECK line. Does that work here?